epub33/core/index.html

```diff
@@ -4092,348 +4097,376 @@ <h4>Prefixed Properties</h4>
+										<td><code>-epub-text-combine: horizontal &lt;number&gt;</code></td>
+										<td>Error</td>
+									</tr>
+								</tbody>
+							
+							</table>
+						</section>
 					</section>
 
 					<section id="sec-css-prefixed-text">
 						<h5>CSS Text Level 3</h5>
-						<table>
-							<thead>
-								<tr>
-									<th>Property</th>
-									<th>Value</th>
-									<th>Mapping to [[CSS-Text-3-20160119]]</th>
-								</tr>
-							</thead>
-							<tbody>
-								<tr>
-									<td>
-										<code>-epub-hyphens</code>
-									</td>
-									<td>
-										<code>none | manual | auto</code>
-									</td>
-									<td>No Change</td>
-								</tr>
-								<tr>
-									<td>
-										<code>-epub-hyphens</code>
-									</td>
-									<td>
-										<code>all</code>
-									</td>
-									<td>Not Supported</td>
-								</tr>
-								<tr>
-									<td>
-										<code>-epub-line-break</code>
-									</td>
-									<td>
-										<code>auto | loose | normal | strict</code>
-									</td>
-									<td>No Change</td>
-								</tr>
-								<tr>
-									<td>
-										<code>-epub-text-align-last</code>
-									</td>
-									<td>
-										<code>auto | start | end | left | right | center | justify</code>
-									</td>
-									<td>No Change</td>
-								</tr>
-								<tr>
-									<td>
-										<code>-epub-word-break</code>
-									</td>
-									<td>
-										<code>normal | keep-all | break-all</code>
-									</td>
-									<td>No Change</td>
-								</tr>
-								<tr>
-									<td>
-										<code>text-transform</code>
-									</td>
-									<td>
-										<code>-epub-fullwidth</code>
-									</td>
-									<td>
-										<code>text-transform: full-width</code>
-									</td>
-								</tr>
-							</tbody>
-						</table>
+						
+						<section id="sec-css-prefixed-text-epub-hyphens">
+						<h6>The <code>-epub-hyphens</code> Property</h6>
+						<p>This is a prefixed version of the <code>hyphens</code> property.</p>
+							<table class="def propdef">
+								<tbody>
+									<tr>
+										<th>Name: </th>
+										<td>
+											<dfn>-epub-hyphens</dfn>
+										</td>
+									</tr>
+									<tr class="value">
+										<th>Value:</th>
+										<td>
+											none | manual | auto | all
+										</td>
+									</tr>
+								</tbody>
+							</table>
+							<p class="note">The <code>auto</code> value is no longer supported in CSS.</p>
+						</section>
+						<section id="sec-css-prefixed-text-epub-line-break">
+						<h6>The <code>-epub-line-break</code> Property</h6>
+						<p>This is a prefixed version of the <code>line-break</code> property.</p>
+							<table class="def propdef">
+								<tbody>
+									<tr>
+										<th>Name: </th>
+										<td>
+											<dfn>-epub-linebreak</dfn>
+										</td>
+									</tr>
+									<tr class="value">
+										<th>Value:</th>
+										<td>
+											auto | loose | normal | strict
```

Comment:
This list isn't exhaustive, which makes sense since 'anywhere' wasn't a value of -epub-line-break. But does that mean it is an error to have -epub-line-break: anywhere? What should reading systems do in that case? Maybe it makes sense to mention that we allow all the values allowed on the unprefixed property?

Also applies to other properties (eg -epub-word-break)

Reply:
Yeah. I think our general principle is that we should allow reading systems to try to do the right thing, even when authors are behaving strangely. So if you see `-epub-line-break: anywhere` the RS can do `line-break: anywhere`.

Reply:
Added a note to that effect at the beginning. There's also the converse question: values that were used when the prefixed properties were created, but that are no longer part of CSS (like `hyphens: all`).